### PR TITLE
Don't try to kill empty list of containers in `integration/runner`

### PR DESCRIPTION
**Pull request by excitoon**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Before:
```
Trying to kill container clickhouse_integration_tests if it's already running
"docker kill" requires at least 1 argument.
See 'docker kill --help'.

Usage:  docker kill [OPTIONS] CONTAINER [CONTAINER...]

Kill one or more running containers
Nothing to kill
```

After:
```
Trying to kill containers name=clickhouse_integration_tests ids=['c1ecddcbd73a']
c1ecddcbd73a
Containers ['c1ecddcbd73a'] killed
```

### CODE REVIEW

**Review by qoega — APPROVED** [review]

File: tests/integration/runner [hunk 1]
```diff
@@ -16,7 +16,7 @@ CURRENT_WORK_DIR = os.getcwd()
 CONTAINER_NAME = "clickhouse_integration_tests"
 
 CONFIG_DIR_IN_REPO = "programs/server"
-INTERGATION_DIR_IN_REPO = "tests/integration"
+INTEGRATION_DIR_IN_REPO = "tests/integration"
```
Comment (alexey-milovidov) [inline tests/integration/runner:19]: :+1:

**Review by alexey-milovidov — APPROVED** [review]

**Comment by tavplubix** [comment]
@excitoon, sorry, I had to revert it, because it has broken integration tests in master for some reason

**Comment by excitoon** [comment]
@tavplubix I can see a potential problem with this code: https://github.com/excitoon-favorites/ClickHouse/blob/91d9a074e15a893e133e3f550c5cd0730870df7a/docker/test/integration/runner/dockerd-entrypoint.sh#L33-L34

If there is no `xargs` it may fail, but though I was not able to run it myself, that should have been very unlikely. How would I test it?

**Comment by tavplubix** [comment]
@excitoon you can find failed checks in master and take a look at reports and logs

**Comment by excitoon** [comment]
Can you attach some link to a broken test result?

**Comment by tavplubix** [comment]
@excitoon you can find failed checks in master and take a look at reports and logs
(but probably it's too late and logs are already cleaned up)

**Pull request merged**